BackofficeBundle/Form/DataTransformer/ChoicesOptionToArrayTransformer.php

```diff
@@ -56,4 +46,27 @@ public function reverseTransform($choices)
 
         return $arrayChoice;
     }
+
+    /**
+     * @param string $choice
+     *
+     * @return string
+     */
+    protected function suppressSpecialCharacterTranslateChoice($choice)
```

Comment:
I still do not agree with the transformation in the same class.

I think that there would be something to do with the class `GenerateIdHelper` of the model-bundle